### PR TITLE
Improvements

### LINKED ISSUE #3: Feat: Add README
Add the following README to the repo: 

# Setup  

1. Install Twilio CLI following [these instructions](https://www.twilio.com/docs/twilio-cli/quickstart) 
1. Install the Infra Plugin for Twilio CLI 
```
$ twilio plugins:install plugin-twilio-infra
```
3. [Download and Install the Pulumi CLI](https://www.pulumi.com/docs/get-started/install/)
4. [Login in to the Pulumi service](https://www.pulumi.com/docs/reference/cli/pulumi_login/). You can either use the Pulumi service backend (you need to sign up to a free account for that) or the local storage, using one of the commands below:
```
# Pulumi Service backend
$ pulumi login

# Local storage
$ pulumi login --local
```
5. If you've never used the Twilio CLI with this project, create a new profile with 
```
$ twilio login
```
6. Login to the project using
```
$ twilio profile:use <profile name>
```
7. Add the following environment variables: 

* `CHAT_SERVICE_SID`
* `FLEX_WORKSPACE_SID`
* `SMS_TASK_CHANNEL_SID` 
* `EVERYONE_TASK_QUEUE_SID`
* `TWILIO_PHONE_NUMBER`
* `FLEX_PROXY_SERVICE_SID`

8. If you don't have an environment defined, use the following command
```
$ twilio infra:environment:new <environment name>
```
9. Deploy the solution using: 
```
$ TWILIO_AUTH_TOKEN=xxxxxx twilio infra:deploy
```
10. In the Twilio [Console Settings page for Flex](https://www.twilio.com/console/flex/settings) make sure that the "Enable Dialpad" is Enabled

# Usage 
1. In Flex click the dialpad button (next to the microhpne on the top right corner). At the bottom of the dialpad you will find the "Outbound SMS" section. 
1. Type the destination number and press on START
1. A new Task / Reservation is generated: Click the accpet button to accept and start chatting

### LINKED ISSUE #4: Bug: Set the proper name for chat channel and task attributes
See https://github.com/vernig/outbound-messaging-iac/pull/1/commits/8168534ef0783ad90beaa8471e2d99c74a5c469e

### PR DESCRIPTION
**Pull request by vernig**
This PR includes: 

* Add a README.md with installation and usage instructions. The instructions provide workaround for https://github.com/twilio-infra-as-code/twilio-pulumi-provider/issues/4 and  https://github.com/twilio-infra-as-code/plugin-twilio-infra/issues/22
* Change `name` attribute for new task created to destination phone number (currently hardcoded to `Kaique Lupo`)
* Change `chatUserFriendlyName` to destination phone number (currently hardcoded to `Kaique Lupo`)

Fix #4 
Fix #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Setup  
+
+1. Install Twilio CLI following [these instructions](https://www.twilio.com/docs/twilio-cli/quickstart)
```
Comment (bermudezmt) [inline README.md]: ``

Comment (bermudezmt) [inline README.md]:
```suggestion
1. [Install the Twilio CLI] (https://www.twilio.com/docs/twilio-cli/quickstart)
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Setup  
+
+1. Install Twilio CLI following [these instructions](https://www.twilio.com/docs/twilio-cli/quickstart) 
+1. Install the Infra Plugin for Twilio CLI
```
Comment (bermudezmt) [inline README.md]: for the Twilio CLI

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+1. Install Twilio CLI following [these instructions](https://www.twilio.com/docs/twilio-cli/quickstart) 
+1. Install the Infra Plugin for Twilio CLI 
+```
+$ twilio plugins:install plugin-twilio-infra
+```
+3. If you've never used the Twilio CLI with this project, create a new profile with 
+```
+$ twilio login
+```
+4. Login to the project using
```
Comment (bermudezmt) [inline README.md]: Log in

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+6. If you don't have an environment defined, use the following command
+```
+$ twilio infra:environment:new <environment name>
+```
+7. Deploy the solution using: 
+```
+$ TWILIO_AUTH_TOKEN=xxxxxx twilio infra:deploy
+```
+8. In the Twilio [Console Settings page for Flex](https://www.twilio.com/console/flex/settings) make sure that the "Enable Dialpad" is Enabled
```
Comment (bermudezmt) [inline README.md]: toggle "Enable Dialpad" to `Enabled`.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+# Usage 
+1. In Flex click the dialpad button (next to the microhpne on the top right corner). At the bottom of the dialpad you will find the "Outbound SMS" section.
```
Comment (bermudezmt) [inline README.md]: In Flex, click the Dialpad 

Correct: microphone

"At the bottom of the Dialpad, you will find the additional "Outbound SMS" text field.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+```
+7. Deploy the solution using: 
+```
+$ TWILIO_AUTH_TOKEN=xxxxxx twilio infra:deploy
+```
+8. In the Twilio [Console Settings page for Flex](https://www.twilio.com/console/flex/settings) make sure that the "Enable Dialpad" is Enabled
+
+# Usage 
+1. In Flex click the dialpad button (next to the microhpne on the top right corner). At the bottom of the dialpad you will find the "Outbound SMS" section. 
+1. Type the destination number and press on START
```
Comment (bermudezmt) [inline README.md]: Enter the destination number and click **Start**.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+1. A new Task / Reservation is generated: Click the accpet button to accept and start chatting
```
Comment (bermudezmt) [inline README.md]: Clicking start generates a new [Task](https://www.twilio.com/docs/flex/admin-guide/core-concepts/routing#tasks). Click **Accept** to accept the incoming task and start a messaging conversation with the customer.

**Review by bermudezmt — CHANGES_REQUESTED** [review]

**Comment by vernig** [comment]
Thanks @bermudezmt for the (thorough) review and the changes. I merged them all. 

@kaiquelupo could you have a look to the PR as well, especially commit 8168534ef0783ad90beaa8471e2d99c74a5c469e? 

Feel free to merge into master if you are happy with it

**Review by kaiquelupo — APPROVED** [review]

**Pull request merged**